### PR TITLE
Fixes #16284: Document reporting protocol options

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/16284

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

File: src/reference/modules/administration/pages/server.adoc [hunk 1]
```diff
@@ -1,5 +1,29 @@
 = Server administration
 
+[[reporting_protocol, Reporting]]
+== Reporting protocol
+
+Reporting data is sent by the agent during each run, and is used by the server to compute compliance.
+
+There are two available protocols, syslog and HTTPS (only with 6.0+ agents). Syslog was used until 6.0 and HTTPS is progressively replacing it.
```
Comment (fanf) [inline src/reference/modules/administration/pages/server.adoc]: syslog (all versions of Rudder) and HTTPS (only with 6.0+ agents)

File: src/reference/modules/administration/pages/server.adoc [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+* No need to interfere with system syslog daemon or to open port 514
+* Allow to check the server certificate, and hence only allow sending reports to the right server
+
+You can switch between the following reporting options at any time:
+
+* syslog: default on upgrades, same as before 6.0.
+* HTTPS with syslog compatibility: keeps compatibility with pre-6.0 agents by using HTTPS reporting for 6.0 agents but still allowing syslog reporting
+* HTTPS: switches to HTTPS only, and disables syslog report forwarding, removing all Rudder-specific log configuration from nodes and servers. This should only be done once all agents are upgraded to 6.0. This allows enforcing report signature and transport through HTTPS.
+
+For now, if you are using the Change only report mode, you cannot use HTTPS reporting yet, and have to stick to syslog for now.
```
Comment (fanf) [inline src/reference/modules/administration/pages/server.adoc]: "for now" duplicated, I would remove the one at the end of the sentence.

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**